OMCompiler/Compiler/Template/MMToJulia.tpl

```diff
@@ -1152,6 +1155,59 @@ let() = Tpl.addSourceTemplateError(errMessage, srcInfo)
 >>
 end error;
 
+template dumpPathJL(Absyn.Path path)
+"Wrapper function for dump path. Needed since certain keywords will have a sligthly different meaning in Julia"
+::=
+match path
+  case FULLYQUALIFIED(__) then
+    '.<%AbsynDumpTpl.dumpPath(path)%>'
+  case QUALIFIED(__) then
+    if (Flags.getConfigBool(Flags.MODELICA_OUTPUT)) then
+    '<%name%>__<%AbsynDumpTpl.dumpPath(path)%>'
+    else
+    '<%name%>.<%AbsynDumpTpl.dumpPath(path)%>'
+  /*Julia keywords have a slightly different semantic meaning*/
+  case IDENT(__) then
```

Comment:
WIP: Basically Julias meaning of the word Real is way more generous then Modelica it means all Integers more or less even Booleans. I also restricted Integer to Signed since Modelica and MetaModelica Integers are Signed